vm.py

```diff
@@ -107,3 +107,12 @@ def poweroff():
   from nagios import schedule_downtime
   execute(schedule_downtime, env['host_string'])
   run("sudo poweroff")
+
+@task
+@hosts('puppetmaster-1.management')
+def host_key(hostname):
+  """Check the SSH host key of a machine"""
+  temp_file = '/tmp/ssh_host_key'
```

Comment:
There could be race condition if two people run this at the same time.

Is it possible to do it without a temporary file? Or store the output of the first command in a variable and then write it to a local temp file using `tempfile.mkstemp()`?